### PR TITLE
chore: Adjust Application update tutorial to match UI onboarding

### PR DESCRIPTION
**Pull request by michaelntech**
Ticket: MEN-6681

# External Contributor Checklist

🚨 Please review the [guidelines for contributing](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md) to this repository.

- [ ] Make sure that all commits follow the conventional commit [specification](https://github.com/mendersoftware/mendertesting/blob/master/commitlint/grammar.md) for the Mender project.

The majority of our contributions are fixes, which means your commit should have
the form below:

```
fix: <SHORT DESCRIPTION OF FIX>

<OPTIONAL LONGER DESCRIPTION>

Changelog: <USER-FRIENDLY-CHANGE-DESCRIPTION> or <None>
Ticket: <TICKET NUMBER> or <None>
```

- [ ] Make sure that all commits are signed with [`git --signoff`](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work). Also note that the signoff author must match the author of the commit.

### Description

Please describe your pull request.

Thank you!

### CODE REVIEW

**Comment by michaelntech** [comment]
To go with https://github.com/mendersoftware/mender-demo-artifact/pull/268 and https://github.com/mendersoftware/gui/pull/3870

File: 01.Get-started/02.Deploy-an-application-update/docs.md [hunk 1]
```diff
@@ -21,74 +21,72 @@ You should have completed one of the following:
...
-use hosted Mender. It contains a small web server you can run on your device.
-Go to the **Releases** tab and select the target release already available.
 
-![connecting a device](Image_5.png)
+In the Device information view for the device you just connected, select **Create a deployment for this device** from the **Device actions**.
 
-Click **CREATE DEPLOYMENT WITH THIS RELEASE** as shown below.
+![create a deployment for this device](Image_5.png)
 
-![create a deployment](Image_6.png)
```
Comment (lluiscampos) [inline 01.Get-started/02.Deploy-an-application-update/docs.md:32]: `Image_6.png` reference has been removed from the guide, let's remove the image altogether.

File: 01.Get-started/02.Deploy-an-application-update/docs.md [hunk 2]
```diff
@@ -21,74 +21,72 @@ You should have completed one of the following:
...
 
-![congratulations server](Image_8.png)
+![create a deployment](Image_8.png)
+
+There is already a demo Artifact available the first time you
+use hosted Mender, for the purposes of this tutorial. It contains a small web server you will run on your device. 
 
-Next step is to **Select a rollout schedule**. You can use the defaults and
-click the **NEXT** button.
+Select the Release with the name beginning *mender-demo-artifact*; this will be deployed to your device.
```
Comment (lluiscampos) [inline 01.Get-started/02.Deploy-an-application-update/docs.md:38]: I think all this is referring to `Image_7`, which is not referenced.

File: 01.Get-started/02.Deploy-an-application-update/docs.md [hunk 3]
```diff
@@ -21,74 +21,72 @@ You should have completed one of the following:
...
+## Step 2 - Modify the application and deploy it
+
 
-The UI tooltips should now take you through modifying the web page you saw in
-[Step 1 - Create a Deployment](#step-1-create-a-deployment). Follow the
-instructions showed on the pop-up and click the **NEXT** button when done.
+Now we'll make an update to the webserver demo running on your device, using a new Release that you will create yourself.
 
-![Upload web page](Image_13.png)
+!!! Leave the demo web page from the previous step open, you'll use it to verify the update at the end of this step.
```
Comment (lluiscampos) [inline 01.Get-started/02.Deploy-an-application-update/docs.md:65]: This is okey to write (and the other comment at the end asking for refresh).

But I think that after the second deployment the GUI still tells you "go to ... and refresh the page to see your update" so the instructions are slightly different.

File: 01.Get-started/02.Deploy-an-application-update/docs.md [hunk 4]
```diff
@@ -21,74 +21,72 @@ You should have completed one of the following:
...
 
-Click the **UPLOAD** button to upload the file that you created in the previous
-step.
+On your workstation, create a new `index.html` file with the simple contents 'Hello World'. This will be the new web page after you update the application, so you'll be able to easily see when your device has received the update. Copy and run the following command to create the file:
 
-![Upload web page](Image_14.png)
```
Comment (lluiscampos) [inline 01.Get-started/02.Deploy-an-application-update/docs.md:75]: Another image de-referenced, please delete it.

**Review by lluiscampos — CHANGES_REQUESTED** [review]
Thanks for refreshing this up @michaelntech!

Once we are here, can we tidy up a bit the images? I found several that are not referenced that should be removed.

File: 01.Get-started/02.Deploy-an-application-update/docs.md (hunk 3 above)
Comment (michaelntech) [reply 01.Get-started/02.Deploy-an-application-update/docs.md:65]: In the updated onboarding we now stop the tooltip instructions after the first deployment, and just refer to the docs for this 2nd optional part - that's why I chose to be explicit here.

Comment (lluiscampos) [reply 01.Get-started/02.Deploy-an-application-update/docs.md:65]: Ok. Thanks!

**Review by lluiscampos — APPROVED** [review]
Great!

**Pull request merged**